rubin_sim/scheduler/utils/utils.py

```diff
@@ -436,49 +440,41 @@ def opsim2obs(self, filename):
 def empty_observation():
     """Return a numpy array that could be a handy observation record
 
-    XXX:  Should this really be "empty visit"? Should we have "visits" made
-    up of multple "observations" to support multi-exposure time visits?
-
-    XXX-Could add a bool flag for "observed". Then easy to track all proposed
-    observations. Could also add an mjd_min, mjd_max for when an observation should be observed.
-    That way we could drop things into the queue for DD fields.
-
-    XXX--might be nice to add a generic "sched_note" str field, to record any metadata that
-    would be useful to the scheduler once it's observed. and/or observationID.
-
     Returns
     -------
     numpy array
```

Comment:
If you want to pedantically follow DM standards, the [return needs to be named](https://developer.lsst.io/python/numpydoc.html#py-docstring-returns).